packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -255,21 +293,29 @@ export class HbarLimitService implements IHbarLimitService {
   private async isDailyBudgetExceeded(
     mode: string,
     methodName: string,
+    txConstructorName: string,
     estimatedTxFee: number = 0,
     requestIdPrefix?: string,
   ): Promise<boolean> {
     if (this.shouldResetLimiter()) {
       await this.resetLimiter();
     }
+    // note: estimatedTxFee is only applicable in a few cases (currently, only for file transactions).
+    //      In most situations, estimatedTxFee is set to 0 (i.e., not considered).
+    //      In such cases, it should still be false if remainingBudget === 0.
     if (this.remainingBudget <= 0 || this.remainingBudget - estimatedTxFee < 0) {
       this.hbarLimitCounter.labels(mode, methodName).inc(1);
       this.logger.warn(
-        `${requestIdPrefix} HBAR rate limit incoming call: remainingBudget=${this.remainingBudget}, totalBudget=${this.totalBudget}, estimatedTxFee=${estimatedTxFee}, resetTimestamp=${this.reset}`,
+        `${requestIdPrefix} Daily HBAR rate limit incoming call: remainingBudget=${this.remainingBudget}, totalBudget=${
+          this.totalBudget
+        }, estimatedTxFee=${estimatedTxFee}, resetTimestamp=${this.reset.getMilliseconds()}, txConstructorName=${txConstructorName} mode=${mode}, methodName=${methodName}`,
       );
       return true;
     } else {
       this.logger.trace(
-        `${requestIdPrefix} HBAR rate limit not reached: remainingBudget=${this.remainingBudget}, totalBudget=${this.totalBudget}, estimatedTxFee=${estimatedTxFee} resetTimestamp=${this.reset}.`,
+        `${requestIdPrefix} Daily HBAR rate limit not reached: remainingBudget=${this.remainingBudget}, totalBudget=${
```

Comment:
Since the process of determining whether the request should be limited involves two flows—checking the daily HBAR limit and assessing user HBAR consumption—I believe that renaming it to `Daily HBAR Rate Limit Incoming Call` would provide a clearer indication that the first scenario has occurred, making the debugging developers quicly point out what outcome they should expected.

Reply:
Limiting is no longer going to be on a daily basis after https://github.com/hashgraph/hedera-json-rpc-relay/pull/3014, just wanted to make sure the logs are correct and this is not missed

Reply:
```suggestion
        `${requestIdPrefix} Total HBAR rate limit not reached: remainingBudget=${this.remainingBudget}, totalBudget=${
```

Reply:
updated.